.travis.yml

```diff
@@ -32,9 +32,13 @@ jobs:
     scripts:
       - bin/run -v
       - bin/run https://github.com/DXHeroes/dx-scanner
+      - bin/run https://github.com/moment/luxon
+      - bin/run https://github.com/intoli/remote-browser
+      - dxs 
     os: windows
   - os: linux
   - stage: release
+    if: os = linux
```

Comment:
This is not necessary. This stage runs just on linux (see line 6). I wanted to allow failure if script run on windows (line 39).